src/components/EditImageModal/index.jsx

```diff
@@ -50,13 +51,13 @@ const initialEditImageModalState = {
   isImageLoading: false,
   isImageDescriptionValid: true,
   isImageDimensionsValid: true,
+  isModalOpen: false,
   isStatusAlertOpen: false,
   imageSource: '',
   imageStyle: '',
-  open: false,
-  currentValidationMessages: {},
   pageNumber: 1,
   shouldShowPreviousButton: false,
+  uploadErrorMessageBody: undefined,
```

Comment:
Also, I wonder if it would make sense to follow the pattern of currentValidationMessages for this as well? So instead of storing the string here, store the WrappedMessage element instead. I think the only benefit would be consistency. I don't know that storing a React element in the state is good or bad. Maybe the string is the way to go? In that case, we might want to just make it consistent either way.

I recall you suggesting the addition of the word "current" to `currentValidationMessages` so it's clear that they're the current messages, so maybe `currentUploadErrorMessage` would make sense here? 

I also think the word "body" kind of throws me off a little, since it makes me think of the modal body, for some reason. Do you think it would be better to keep the variable name unaware of where it's being used, in case it gets used elsewhere - kind of like the validation messages.

Reply:
Changing this to `currentUploadErrorMessage` sounds good. I'll double check the change to `null` and any side affects that might arise.